I'm having trouble to get the tests running. I wanted to try to write some additional tests for the other PR I'm preparing. 
I've tried to follow the instructions by installing the requirements via
```
$ python3.11 -m pip install -r conans/requirements_server.txt
$ python3.11 -m pip install -r conans/requirements_dev.txt
```
and I've exported the env variable
```
$ export PYTHONPATH=$PYTHONPATH:$(pwd)
```
I've also created a `conftest_user.py` with the following contents:
```
tools_locations = {
    'svn': {"disabled": True},
    'cmake': {"disabled": True},
    'ninja': {
        "disabled": True
    },
    'meson': {"disabled": True},
    'bazel':  {
        "disable": True,
    },
    'autotools': {"disabled": True},
    'premake': {"disabled": True},
    'android_ndk': {"disabled": True},
}
```

But when try to run the command I get:
```
python3.11 -m pytest .
============================================================ test session starts ============================================================
platform darwin -- Python 3.11.4, pytest-6.2.5, py-1.11.0, pluggy-1.2.0
rootdir: /Users/picard/Developer/conan-io, configfile: pytest.ini
plugins: xdist-3.3.1
collected 0 items / 1 error

================================================================== ERRORS ===================================================================
_______________________________________________________ ERROR collecting test session _______________________________________________________
/opt/homebrew/Cellar/python@3.11/3.11.4/Frameworks/Python.framework/Versions/3.11/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1204: in _gcd_import
    ???
<frozen importlib._bootstrap>:1176: in _find_and_load
    ???
<frozen importlib._bootstrap>:1126: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:241: in _call_with_frames_removed
    ???
<frozen importlib._bootstrap>:1204: in _gcd_import
    ???
<frozen importlib._bootstrap>:1176: in _find_and_load
    ???
<frozen importlib._bootstrap>:1126: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:241: in _call_with_frames_removed
    ???
<frozen importlib._bootstrap>:1204: in _gcd_import
    ???
<frozen importlib._bootstrap>:1176: in _find_and_load
    ???
<frozen importlib._bootstrap>:1147: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:690: in _load_unlocked
    ???
<frozen importlib._bootstrap_external>:940: in exec_module
    ???
<frozen importlib._bootstrap>:241: in _call_with_frames_removed
    ???
conans/__init__.py:1: in <module>
    from conans.client.build.autotools_environment import AutoToolsBuildEnvironment
conans/client/build/autotools_environment.py:6: in <module>
    from conans.client.build.compiler_flags import (architecture_flag, build_type_define,
conans/client/build/compiler_flags.py:13: in <module>
    from conans.client.tools.apple import to_apple_arch
conans/client/tools/__init__.py:4: in <module>
    from .apple import *
conans/client/tools/apple.py:5: in <module>
    from conans.util.runners import check_output_runner
conans/util/runners.py:9: in <module>
    from conans.client.tools.files import load
conans/client/tools/files.py:12: in <module>
    from patch_ng import fromfile, fromstring
E   ModuleNotFoundError: No module named 'patch_ng'
========================================================== short test summary info ==========================================================
ERROR  - ModuleNotFoundError: No module named 'patch_ng'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
============================================================= 1 error in 0.10s ==============================================================
```
same if try to run with `sudo`.

I get a similar error if I try to run only the apple tools tests:
```
python3.11 -m pytest conans/test/unittests/tools/apple/test_apple_tools.py -s
ImportError while loading conftest '/Users/picard/Developer/conan-io/conans/test/conftest.py'.
conans/__init__.py:1: in <module>
    from conans.client.build.autotools_environment import AutoToolsBuildEnvironment
conans/client/build/autotools_environment.py:6: in <module>
    from conans.client.build.compiler_flags import (architecture_flag, build_type_define,
conans/client/build/compiler_flags.py:13: in <module>
    from conans.client.tools.apple import to_apple_arch
conans/client/tools/__init__.py:4: in <module>
    from .apple import *
conans/client/tools/apple.py:5: in <module>
    from conans.util.runners import check_output_runner
conans/util/runners.py:9: in <module>
    from conans.client.tools.files import load
conans/client/tools/files.py:12: in <module>
    from patch_ng import fromfile, fromstring
E   ModuleNotFoundError: No module named 'patch_ng'
```

Any help would be appreciated.

--- a/conans/test/integration/toolchains/apple/test_xcodedeps.py
+++ b/conans/test/integration/toolchains/apple/test_xcodedeps.py
@@ -9,7 +9,7 @@ from conans.test.integration.toolchains.apple.test_xcodetoolchain import _get_fi
 from conans.test.utils.tools import TestClient
 
 _expected_dep_xconfig = [
-    "HEADER_SEARCH_PATHS = $(inherited) $(HEADER_SEARCH_PATHS_{name}_{name})",
+    "SYSTEM_HEADER_SEARCH_PATHS = $(inherited) $(SYSTEM_HEADER_SEARCH_PATHS_{name}_{name})",
     "GCC_PREPROCESSOR_DEFINITIONS = $(inherited) $(GCC_PREPROCESSOR_DEFINITIONS_{name}_{name})",
     "OTHER_CFLAGS = $(inherited) $(OTHER_CFLAGS_{name}_{name})",
     "OTHER_CPLUSPLUSFLAGS = $(inherited) $(OTHER_CPLUSPLUSFLAGS_{name}_{name})",
@@ -19,7 +19,7 @@ _expected_dep_xconfig = [
 ]
 
 _expected_conf_xconfig = [
-    "HEADER_SEARCH_PATHS_{name}_{name}[config={configuration}][arch={architecture}][sdk={sdk}{sdk_version}] = ",
+    "SYSTEM_HEADER_SEARCH_PATHS_{name}_{name}[config={configuration}][arch={architecture}][sdk={sdk}{sdk_version}] = ",
     "GCC_PREPROCESSOR_DEFINITIONS_{name}_{name}[config={configuration}][arch={architecture}][sdk={sdk}{sdk_version}] = ",
     "OTHER_CFLAGS_{name}_{name}[config={configuration}][arch={architecture}][sdk={sdk}{sdk_version}] = ",
     "OTHER_CPLUSPLUSFLAGS_{name}_{name}[config={configuration}][arch={architecture}][sdk={sdk}{sdk_version}] = ",

